extensions/wikibase/src/org/openrefine/wikibase/qa/scrutinizers/ItemRequiresScrutinizer.java

```diff
@@ -100,20 +106,32 @@ public void scrutinizeStatementEdit(StatementEntityEdit update) {
                 PropertyIdValue itemRequiresPid = constraint.itemRequiresPid;
                 List<Value> itemList = constraint.itemList;
                 if (!propertyIdValueValueMap.containsKey(itemRequiresPid)) {
-                    QAWarning issue = new QAWarning(update.isNew() ? newItemRequirePropertyType : existingItemRequirePropertyType,
+                    QAWarning issue = new QAWarning(update.isNew()
+                            ? (constraint.itemList.size() == 1 ? newItemRequirePropertywithSuggestedValueType : newItemRequirePropertyType)
+                            : (constraint.itemList.size() == 1 ? existingItemRequirePropertywithSuggestedValueType
+                                    : existingItemRequirePropertyType),
```

Comment:
I wonder if `constraint.itemList.size() == 1` can be true at all in this case, because we are in an `if (!propertyIdValueValueMap.containsKey(itemRequiresPid)) { … }`
It looks like in this case, the constraint definition does not include any items as expected values, so we should know that `constraint.itemList.size() == 0`, no? Or did I miss anything?

If that's indeed true, then that means there are two less warning ids to introduce (and we can remove the associated translations).

Reply:
@wetneb In my testing using the test data, I see a warning of type "existing-item-requires-certain-other-statement" P633 in relation to P1435 has 16 suggested values.

Reply:
This one is expected indeed, but I don't think you can generate `new-item-requires-certain-other-statement-with-suggested-value` or `existing-item-requires-certain-other-statement-with-suggested-value`.

Reply:
"ew-item-requires-certain-other-statement-with-suggested-value" - comes up in the unit test "testNewItemTrigger"

Reply:
It comes up in the condition of an `if` block, but nothing tells me that this condition is ever met ;)

Reply:
Fixed the miss in the unit test.